example/lib/example.g.dart

```diff
@@ -15,7 +15,7 @@ Person _$PersonFromJson(Map<String, dynamic> json) {
     lastOrder: json['last-order'] == null
         ? null
         : DateTime.parse(json['last-order'] as String),
-    orders: (json['orders'] as List)
+    orders: (json['orders'] as List<dynamic>)
```

Comment:
FYI, we are skipping the `is` and `as` checks on strict-raw-types internally at Google. 99% of the time, you're looking at unstructured parsed data, like this JSON, and your only choice is to write `as List<dynamic>`, and you've neither improved readability nor type promotion. :/ In analysis_options.yaml you'd ignore `STRICT_RAW_TYPE_IN_AS` and `STRICT_RAW_TYPE_IN_IS`.

Reply:
If we are not using them internally and recommending against them externally - should they be dropped altogether?

Reply:
We're not using them internally because it looks like they add more noise than benefit, and thats a lot to _push_ onto developers. But I can see a dev finding these useful on their own individual pub package, and wanting the checks.

I'm not recommending against them externally; I'm just mentioning what we do internally, in case it is useful information.

Reply:
> I'm just mentioning what we do internally, in case it is useful information.

Honestly, that seems like pretty good data that they're not worth their weight.